sechub-administration/src/main/java/com/mercedesbenz/sechub/domain/administration/project/ProjectData.java

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.domain.administration.project;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ProjectData {
+    public static final String PROPERTY_PROJECT_ID = "projectId";
+    public static final String PROPERTY_OWNER = "owner";
+    public static final String PROPERTY_IS_OWNED = "isOwned";
+    public static final String PROPERTY_ASSIGNED_USERS = "assignedUsers";
+
+    private String projectId;
+    private String owner;
+    private Boolean isOwned;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
here you put the annotation on the field while you put the other annotations on the getters
while both approaches work I would not mix them 

consider putting all annotations on one level (either field or getter)